subprojects/configuration-cache/build.gradle.kts

```diff
@@ -79,6 +79,7 @@ dependencies {
     implementation(project(":build-option"))
 
     implementation(libs.asm)
+    implementation(libs.ant)
```

Comment:
This thing is bit concerning me, because I wouldn't add new dependency without need. Alternatively I could create wrapper/interface around DirectoryScanner, but this is a bit error prone IMO

Reply:
I don't think there's anything particularly bad with this dependency. Ant is part of the Gradle runtime regardless.